README.md

```diff
@@ -168,7 +168,7 @@ athena:
 
 - `external_location` (`default=none`)
   - If set, the full S3 path in which the table will be saved.
-  - Does not work with Iceberg table or Hive table with `ha` set to true.
+  - Does not work with Hive table with `ha` set to true.
```

Comment:
this need to be rephrased, external_location works only if materialization is incremental for iceberg.